src/sql/workbench/services/notebook/browser/sql/sqlSessionManager.ts

```diff
@@ -426,7 +430,7 @@ export class SQLFuture extends Disposable implements FutureInternal {
 			channel: 'shell',
 			type: 'execute_reply',
 			content: {
-				status: 'ok',
+				status: this._errorOccurred && this._stopOnError ? 'error' : 'ok',
```

Comment:
I think it should be this._errorOccurred || this._stopOnError ?

Reply:
Hmm I only want to stop when the option is set and an error occurs. Did I mess it up?

Reply:
if either of those are true, it still returns 'ok', is that what you want?

Reply:
@MaddyDev - That's not correct. It'll evaluate the bitwise AND first and then do the conditional 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence

https://typescript-play.js.org/#code/MYewdgziA2CmB00QHMAUAzAhtCsAEAZAXljvgPx4DksATrSLVXgFzUgDWVAlANx4B6AXk4AoUJBgIkaAC60ArviIlsuPJRr1GzNlU49+QkR3HgocRClTylhYrYrU6DJq3Zc+g4S8ZA